<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMark</class>
 <widget class="QWidget" name="FormMark">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>882</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QPushButton" name="pushButtonUpd">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Обновить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DbViewer" name="tableViewMark">
         <property name="selectionMode">
          <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Проволока</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Тип по AWS</string>
           </property>
          </widget>
         </item>
         <item row="3" column="5">
          <widget class="QCheckBox" name="checkBoxKat">
           <property name="text">
            <string>Каталог</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Знаменатель</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Тип по ISO</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="DbComboBox" name="comboBoxISO">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="DbComboBox" name="comboBoxVid">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="DbComboBox" name="comboBoxAWS">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Режим повт. прок.</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditMark"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Марка</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Название для серт.</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLineEdit" name="lineEditPr"/>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="lineEditMarkSert"/>
         </item>
         <item row="4" column="5">
          <widget class="QLineEdit" name="lineEditInd"/>
         </item>
         <item row="4" column="4">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Индекс сортировки</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Вид</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEditVl"/>
         </item>
         <item row="0" column="5">
          <widget class="DbComboBox" name="comboBoxGroup">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Букв. обозн</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="DbComboBox" name="comboBoxGost">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Полож. при сварке</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Тип по ГОСТ</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Группа</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="DbComboBox" name="comboBoxProv">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="3" colspan="2">
          <widget class="DbComboBox" name="comboBoxPol">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="DbComboBox" name="comboBoxBukv">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Описание для каталога</string>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="DbComboBox" name="comboBoxZnam">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Доп. сод. влаги, %</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_21">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Особые свойства</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="4">
          <widget class="QPlainTextEdit" name="plainTextEditDescrCat">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>70</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="5" column="5">
          <widget class="QLabel" name="labelPix">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QPlainTextEdit" name="plainTextEditDescrSpec">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="label_22">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Особенности сварки</string>
           </property>
          </widget>
         </item>
         <item row="6" column="3" colspan="3">
          <widget class="QPlainTextEdit" name="plainTextEditDescrFeature">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>Упаковка, штрихкод</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="DbViewer" name="tableViewPack">
             <property name="minimumSize">
              <size>
               <width>550</width>
               <height>0</height>
              </size>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>Характеристики плавления</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="DbViewer" name="tableViewPlav"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QLabel" name="label_30">
             <property name="text">
              <string>Доп. проволока</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="DbViewer" name="tableViewProvol"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxVar">
         <property name="title">
          <string>Параметры варианта</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>Вариант</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="DbComboBox" name="comboBoxVar">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonEdtVar">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Редактировать варианты</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="res.qrc">
                <normaloff>:/images/key.png</normaloff>:/images/key.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonCreVar">
              <property name="text">
               <string>Создать</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonSaveVar">
              <property name="text">
               <string>Сохранить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonDelVar">
              <property name="text">
               <string>Удалить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonCopy">
              <property name="text">
               <string>Заполнить</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEditVarZnam">
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEditProcVar">
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_26">
                <property name="maximumSize">
                 <size>
                  <width>130</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Знаменатель</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_16">
                <property name="maximumSize">
                 <size>
                  <width>130</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Режим повт. прок.</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0" colspan="2">
               <widget class="DbViewer" name="tableViewInt">
                <property name="minimumSize">
                 <size>
                  <width>220</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_27">
                <property name="text">
                 <string>Описание</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="plainTextEditDescr">
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QLabel" name="label_19">
                <property name="text">
                 <string>Токи</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="DbViewer" name="tableViewAmp">
                <property name="minimumSize">
                 <size>
                  <width>350</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <item>
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>Нормативная документация</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="DbViewer" name="tableViewGost"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Химия</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="DbViewer" name="tableViewChem"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>Механика</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="DbViewer" name="tableViewMech"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutLabel">
         <item>
          <widget class="QLabel" name="label_28">
           <property name="text">
            <string>Дата этикетки:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DbDateEdit" name="dateEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_29">
           <property name="text">
            <string>Диам.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DbComboBox" name="comboBoxDiam"/>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxOrder">
           <property name="text">
            <string>Заказ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cmdLbl">
           <property name="text">
            <string>Этикетка</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutMap">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="cmdExt">
           <property name="text">
            <string>Выгрузить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DbViewer</class>
   <extends>QTableView</extends>
   <header>db/dbviewer.h</header>
  </customwidget>
  <customwidget>
   <class>DbDateEdit</class>
   <extends>QDateEdit</extends>
   <header>db/dbdateedit.h</header>
  </customwidget>
  <customwidget>
   <class>DbComboBox</class>
   <extends>QComboBox</extends>
   <header>db/dbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableViewMark</tabstop>
  <tabstop>lineEditMark</tabstop>
  <tabstop>lineEditMarkSert</tabstop>
  <tabstop>comboBoxGroup</tabstop>
  <tabstop>comboBoxVid</tabstop>
  <tabstop>comboBoxProv</tabstop>
  <tabstop>comboBoxGost</tabstop>
  <tabstop>comboBoxISO</tabstop>
  <tabstop>comboBoxAWS</tabstop>
  <tabstop>comboBoxBukv</tabstop>
  <tabstop>comboBoxPol</tabstop>
  <tabstop>lineEditVl</tabstop>
  <tabstop>lineEditPr</tabstop>
  <tabstop>lineEditInd</tabstop>
  <tabstop>tableViewChem</tabstop>
  <tabstop>tableViewMech</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
